Add locust tests for titiler-cmr

maxrjones opened the pull request:
Adds a new notebook and locustfile that tests titiler-cmr using https://dev-titiler-cmr.delta-backend.com

maxrjones commented:
the units are wrong on the last figure, I need to update to (ms)

abarciauskas-bgse commented on 02-run-tests/06-cmr-netcdf.ipynb:
```diff
@@ -61,11 +61,11 @@
       "application/vnd.holoviews_exec.v0+json": "",
```
<pre class="ql-syntax" spellcheck="false">Line #1.    def load_earthaccess_data(concept_id, cmr_query):
</pre><p>Do you think we should move these functions, all except maybe <code>run_cmr_tile_tests</code> to .py files to simplify this notebook? I could imagine actually that the first 4 lines of <code>run_cmr_tile_test</code> are a part of the CMRTileTest initialization, but maybe the <code>get_*</code> functions go in a different file as they are not specific to CMR.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/tile-benchmarking/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

abarciauskas-bgse replied:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Search and open dataset associated with concept id and time range arguments using earthaccess"""
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/tile-benchmarking/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

abarciauskas-bgse replied:
<pre class="ql-syntax" spellcheck="false">Line #16.    def get_extent(ds):
</pre><p>Nit but maybe <code>get_webmercator_extent</code> is a better name? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/tile-benchmarking/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

abarciauskas-bgse commented:
Thanks @maxrjones I made some comments on the notebooks but none of them are blocking. Perhaps some functions from 01-cmr-netcdf.ipynb could also be put in helper files to simplify the notebook, I think the decision is yours, I just thought it might help for readability in a quarto book to keep the notebooks which run tests and display results as simple as possible.

maxrjones commented:
Thanks for the review Aimee! I addressed all your comments except "Should this URI be programmatically generated using datetime.now?" because it's not certain that the "local" tests and the locust tests will be run at the same time. It would probably be helpful to use fsspec to list available tests to run programmatically, which I will add as a potential TODO.
